src/main/java/mate/academy/service/impl/UserServiceImpl.java

```diff
@@ -26,7 +26,8 @@ public User save(User user) {
 
     @Override
     public User findById(Long id) {
-        return userDao.findById(id).get();
+        return userDao.findById(id)
+            .orElseThrow(() -> new RuntimeException("Can't get user by id: " + id));
```

Comment:
Why not DataProcessingException?

Reply:
it's an exception was created only for DAO layer, so it's better to use other exception on service layer